launch/app/app.bndrun

```diff
@@ -65,7 +63,8 @@ feature.openhab-model-runtime-all: \
 	bnd.identity;id='org.openhab.ui.basic',\
 	bnd.identity;id='org.openhab.ui.iconset.classic',\
 	bnd.identity;id='org.apache.aries.jax.rs.whiteboard',\
-	bnd.identity;id='org.ops4j.pax.web.pax-web-extender-whiteboard'
+	bnd.identity;id='org.ops4j.pax.web.pax-web-extender-whiteboard',\
+	bnd.identity;id='org.openhab.binding.netatmo'
```

Comment:
```suggestion
```

Reply:
Shouldn't the netatmo binding be kept our of here completely? It wasn't present before, and I guess @clinique only added it because he happens to be working on it.

Reply:
Yes that's why I added the suggestion. :slightly_smiling_face:

Reply:
:facepalm: I should have paid more attention to it (was looking at it from 3rd party client, not GitHub UI).